app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -65,4 +77,98 @@ describe('Entry Component', () => {
       expect(screen.queryByTestId('global-spinner')).not.toBeInTheDocument();
     });
   });
+
+  it('should display the header with the league name, survivors, and week number, without a past weeks link', async () => {
```

Comment:
Correct me if I'm wrong, but it seems like these tests are doing the same thing, no? Seems redundant to me, unless that was the intention.

Reply:
> Correct me if I'm wrong, but it seems like these tests are doing the same thing, no? The only difference I see is week number. Seems redundant to me, unless that was the intention.

@choir27 The tests are different as when the week number is greater than one a new link should appear to view the past week's entries. The first test is testing week one, while the second test is testing week two and onwards, so that the additional link should be there. 
![Arc UIUX re-implement league entries UI-000134](https://github.com/user-attachments/assets/3a131286-4d63-45cb-bbd7-8abd27a64727)